src/cmdlib.sh

```diff
@@ -254,13 +254,13 @@ EOF
         echo 'ref: "'"${ref}"'"' >> "${override_manifest}"
     fi
     if [ -d "${configdir}/overlay" ]; then
+        echo -n "Committing ${configdir}/overlay... "
+        ostree commit --repo="${tmprepo}" --tree=dir="${configdir}/overlay" -b "${name}-overlay" \
```

Comment:
Bikeshed, how about `${name}-config-overlay` now that we have multiple overlay types?

Reply:
Updated! :arrow_down: